Cargo.toml

```
@@ -31,6 +31,7 @@ members = [
 
 [workspace.lints.clippy]
 type_complexity = "allow"
+doc_markdown = "forbid"
```

```suggestion
doc_markdown = "warning"
```
Could you keep it as warning?

---

yep! does that mean i should it add it back to the ci tool so that warnings break the lint execution?

---

Nah, it'll get caught in CI.